test/evergreen.yml

```diff
@@ -5584,3 +5584,69 @@ buildvariants:
       batchtime: 10080 # 7 days
     - name: s-string
       batchtime: 129600 # 90 days
+
+- name: ubuntu2004-release
+  display_name: "~ Ubuntu 20.04 (Release Build Type)"
+  run_on:
+  - ubuntu2004-small
+  batchtime: 720 # 12 hours
+  expansions:
+    CMAKE_BUILD_TYPE: -DCMAKE_BUILD_TYPE=Release
+    test_env_vars:
+      WT_TOPDIR=$(git rev-parse --show-toplevel)
+      WT_BUILDDIR=$WT_TOPDIR/cmake_build
+      LD_LIBRARY_PATH=$WT_BUILDDIR
+  tasks:
+    - name: compile
```

Comment:
This test list LGTM. 
On Jira there's also mention of a Release variant for stress testing. Did we make a decision on whether we want one?